package.json

```diff
@@ -18,6 +18,8 @@
     "test": "jest --config=./jest.conf.json",
     "test-update": "jest --config=./jest.conf.json --updateSnapshot",
     "test-cypress": "cypress open",
+    "cypress:react": "npx cypress open-ct",
```

Comment:
I prefer the `:` syntax that we have here, but we should update the other commands to match in a different PR.

Reply:
done